src/CoreText/CTFontDescriptor.cs

```diff
@@ -155,207 +155,231 @@ public CTFontDescriptorAttributes ()
-				return !value.HasValue ? null : (CTFontOrientation?) value.Value;
+				if (CTFontDescriptorAttributeKey.FontOrientation is NSString fontOrientation) {
+					var value = Adapter.GetUInt32Value (Dictionary, fontOrientation);
+					return !value.HasValue ? null : (CTFontOrientation?) value.Value;
+				}
+				return null;
 			}
 			set {
-				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.FontOrientation, 
+				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.FontOrientation!, 
 						value.HasValue ? (uint?) value.Value : null);
 			}
 		}
 
 		public CTFontFormat? FontFormat {
 			get {
-				var value = Adapter.GetUInt32Value (Dictionary, CTFontDescriptorAttributeKey.FontFormat);
-				return !value.HasValue ? null : (CTFontFormat?) value.Value;
+				if (CTFontDescriptorAttributeKey.FontFormat is NSString fontFormat) {
+					var value = Adapter.GetUInt32Value (Dictionary, fontFormat);
+					return !value.HasValue ? null : (CTFontFormat?) value.Value;
+				}
+				return null;
 			}
 			set {
-				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.FontFormat, 
+				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.FontFormat!, 
 						value.HasValue ? (uint?) value.Value : null);
 			}
 		}
 
 		public CTFontManagerScope? RegistrationScope {
 			get {
-				var value = Adapter.GetUnsignedIntegerValue (Dictionary, CTFontDescriptorAttributeKey.RegistrationScope);
-				return !value.HasValue ? null : (CTFontManagerScope?) (ulong) value.Value;
+				if (CTFontDescriptorAttributeKey.RegistrationScope is NSString registrationScope) {
+					var value = Adapter.GetUnsignedIntegerValue (Dictionary, registrationScope);
+					return !value.HasValue ? null : (CTFontManagerScope?) (ulong) value.Value;
+				}
+				return null;
 			}
 			set {
-				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.RegistrationScope,
+				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.RegistrationScope!,
 				value.HasValue ? (nuint?) (ulong) value.Value : null);
 			}
 		}
 
 		public CTFontPriority? Priority {
 			get {
-				var value = Adapter.GetUInt32Value (Dictionary, CTFontDescriptorAttributeKey.Priority);
-				return !value.HasValue ? null : (CTFontPriority?) value.Value;
+				if (CTFontDescriptorAttributeKey.Priority is NSString priority) {
+					var value = Adapter.GetUInt32Value (Dictionary, priority);
+					return !value.HasValue ? null : (CTFontPriority?) value.Value;
+				}
+				return null;
 			}
 			set {
-				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.Priority, 
+				Adapter.SetValue (Dictionary, CTFontDescriptorAttributeKey.Priority!, 
 						value.HasValue ? (uint?) value.Value : null);
 			}
 		}
 
 		public bool Enabled {
 			get {
 				var value = (NSNumber) Dictionary [CTFontDescriptorAttributeKey.Enabled];
-				if (value is null)
-					return false;
-				return value.Int32Value != 0;
+					if (value is null)
+						return false;
+					return value.Int32Value != 0;
```

Comment:
This seems like an indentation change only?

Reply:
Whoops, I'm on it!